arangodb-net-standard.Test/CursorApi/CursorApiClientTest.cs

```diff
@@ -159,6 +163,20 @@ public async Task PostCursorAsync_ShouldThrow_WhenAqlIsNotValid()
             Assert.Equal(1203, ex.ApiError.ErrorNum);
         }
 
+        [Fact]
+        public async Task PostCursorAsync_ShouldThrow_WhenAqlIsNotValid2()
```

Comment:
I left a comment on the issue. This is a valid query, so this test doesn't set the correct expectation.